src/components/SearchResults/ElectronicResourcesLink.tsx

```diff
@@ -17,8 +17,12 @@ const ElectronicResourcesLink = ({
   electronicResources,
 }: ElectronicResourcesLinkProps) => {
   return (
-    <Box mt="s" mb="s">
-      <Text mb="xxs" fontSize="desktop.body.body1" fontWeight="bold">
+    <Box mt="s">
+      <Text
+        mb="xxs"
+        fontSize={{ base: "mobile.body.body1", md: "desktop.body.body1" }}
```

Comment:
@bigfishdesign13 just noticed that the "default" or "body" values for the `size` prop do not use this font size value. Should we update this in the DS?

Reply:
Yeah I noticed that the body1 and body2 variants in the Text component don't have the appropriate values set. We should fix that on the DS level so we can specify a variant and remove these overrides.

Maybe we can keep these for now and add TODO comments?